### PR TITLE
word-tokenization: Lookup definitions & images without punctuation

### PR DESCRIPTION
**Pull request by antoviaque**
Uses nltk tokenization to split words, and then remove punctuation
tokens. Stores clean list of words in `SearchAPIView.words`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,3 +34,9 @@ build:
 	#   catching errors here.
 	${UGLIFYJS} -c -m -o static/build/js/vendor/require.js \
 	    static/build/js/vendor/require.js
+
+update-nltk:
+	@echo Downloading corpus files for nltk library...
+	python -m nltk.downloader punkt
```
Comment (talos) [inline Makefile:40]: I'm not sure what version of nltk we're using -- could you please add this to `requirements.txt`?

**Comment by talos** [comment]
Excepting updating `requirements.txt`, this looks good to me.  I'm going to assume you're using the most recent version (worked for me) and merge this in.

File: Makefile (hunk 1 above)
Comment (antoviaque) [inline Makefile:40]: Ups, sorry - forgot to do a `pip freeze`. Done.

**Comment by talos** [comment]
whoops! ya got it.  thankee :)

**Pull request merged**